tests/antares/services/api_services/test_study_api.py

```diff
@@ -472,3 +474,36 @@ def test_read_outputs(self):
             mocker.get(run_url, json={"description": error_message}, status_code=404)
             with pytest.raises(OutputsRetrievalError, match=error_message):
                 self.study.read_outputs()
+
+            # ==== get_matrix() ====
```

Comment:
You should probably put this in a separated test as you're not really checking the same thing as the first part of the test and you can re-introduce a fake output at the start of your test